content/html/concepts/tags/terms/img/img.md

```diff
@@ -35,8 +35,11 @@ Below is an example of an image wrapped in an anchor tag which creates an image
   <head> </head>
   <body>
     <a href="https://codecademy.com">
-      <img src="logo.png" alt="Codecademy Home Page" width="256" height="64" />
+      <img src="cc_logo.png" alt="Codecademy Home Page" width="256" height="64" />
     </a>
   </body>
 </html>
 ```
+This is what the rendered image look like:
+
+[Codecademy logo](![Codecademy logo](../../../../../../media/cc_logo.png)
```

Comment:
```suggestion
![Codecademy logo](https://raw.githubusercontent.com/Codecademy/docs/main/media/cc_logo.png)
```

Reply:
Relative paths to files in the `media/` folder will ultimately not render in the live entries.